src/test/kotlin/com/exactpro/th2/check1/rule/TestChain.kt

```diff
@@ -70,9 +74,9 @@ class TestChain: AbstractCheckTaskTest() {
 
     @Test
     fun `sequence rules - two succeed`() {
-        val streams = createStreams(messages = (1..4).map(::createMessage))
+        val streams = createStreams(messages = (0..4).map(::createMessage))
```

Comment:
Why have you changed the sequence?

Reply:
Because we will receive an additional event with the failed status, which will notify about the missed start message in the stream

Reply:
The checkpoint does not work properly then. We don't pass the checkpoint here at all. So why does it reports about missing start message if we did not set one?